server/src/main/java/org/elasticsearch/cluster/coordination/CoordinationState.java

```diff
@@ -480,6 +480,7 @@ default void markLastAcceptedStateAsCommitted() {
                     metaDataBuilder = MetaData.builder(lastAcceptedState.metaData());
                 }
                 metaDataBuilder.clusterUUIDCommitted(true);
+                logger.info("cluster UUID [{}]", lastAcceptedState.metaData().clusterUUID());
```

Comment:
I wonder if this should say "cluster UUID set to [{}]" to differentiate from the other log message, making it clear that here we snap into the UUID.

Reply:
ok; it's a different logger, but I can see that being overlooked.